.github/workflows/ci.yml

```diff
@@ -232,10 +251,15 @@ jobs:
     - name: Checkout Code
       uses: actions/checkout@v3
 
+    - name: Get CMake and Ninja
+      uses: lukka/get-cmake@latest
+      with:
+        cmakeVersion: ${{ runner.os == 'Windows' && '3.24' || '3.22' }} # Set to minimum required version (3.24 on Windows and 3.22 elsewhere) as specified in CMakeLists.txt to test compatibility
+        ninjaVersion: latest
```

Comment:
We don't really need to use the lowest possible CMake when configuring for clang-tidy but at least this means avoiding downloading Ninja again which saves about 20 seconds.